README.md

```diff
@@ -1,61 +1,131 @@
-[![Juju logo](doc/juju-logo.png?raw=true)](https://jujucharms.com/)
-
-Simple, secure and stable devops tooling. 
-Juju keeps complexity low and productivity high. 
-Built to manage today’s complex application architectures wherever they are run.
- 
-Built for
-- SRE and operations teams
-- Developers
-- Data engineers
-
-Excels at
-- Making your deployment understandable 
-- Simplifying post-install operations, such as upgrades, updates and configuration management
-- Managing hybrid-cloud services, whether on Kubernetes, VMs, bare metal or any combination
-
-## Why Juju?
-
-Juju increases your productivity and decreases your costs.
-
-- **Increase confidence**  
- If you have ever put off upgrading something in production because something might break, then consider Juju.
- Juju allows applications to automatically negotiate their configuration, creating optimal settings dynamically.
-
-
-- **Reduce complexity**  
- Microservices have made applications simpler, but operations more complex.
- Regain your understanding of the whole stack.
-
-
-- **Strengthen operations knowledge**  
- Everyone has their in-house expert.
- Encapsulating their know-how in charms distributes that knowledge throughout the business.
-
-
-- **Simplify day two and beyond**  
- Upgrades, provisioning new capacity, applying configuration changes can be subtle and difficult.
- Juju takes responsibility for them, as well as deployment.
-
-
- - **Maintain portability and repeatability**  
- Retain control over your deployment and eliminate the need for vendor-specific offerings.
- Your devops tooling should be cloud-agnostic and Kubernetes-aware.
-
+[![Juju logo](doc/juju-logo.png?raw=true)](https://juju.is/)
+
+[Juju is a model-driven **Operator Lifecycle Manager**
+(OLM)](https://juju.is/overview). Juju greatly improves the experience of
```

Comment:
Perhaps

Juju is a model-driven Operator Lifecycle Manager (OLM), which greatly improves ...

ie avoid 2 consecutive sentences starting with "Juju".

Reply:
Yes, but I am also hopeful that the more pedantic approach is better for "what is..." type search queries.